python/CMakeLists.txt

```diff
@@ -1,21 +1,23 @@
 # ~~~
 # This file is part of the dune-gdt project:
-#   https://github.com/dune-community/dune-gdt
+# https://github.com/dune-community/dune-gdt
```

Comment:
Just saw this by chance. Looks like somethings off with our license header script. Ideas, @renefritze?

Reply:
I'm afraid that's my VS Code that thinks it's fixing markdown headings or some such.

Reply:
pylicense will fix that again though